platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ImportMigrationContextImpl.java

```diff
@@ -149,6 +152,20 @@ public ImportMigrationEntry getEntry(Path path) {
     return entries(path).filter(e -> filter.matches(e.getPath()));
   }
 
+  @Override
+  public void setImportedSystemProperties(Properties props) {
```

Comment:
❗️ Should not be set this way. This properties object should be set at construction time and be marked `final`.